extension/data/tbapi.js

```diff
@@ -338,12 +339,13 @@
      * @param {string} cssClass The flair's CSS class
      * @returns {Promise}
      */
-    TBApi.flairUser = (user, subreddit, text, cssClass) => TBApi.post('/api/flair', {
+    TBApi.flairUser = (user, subreddit, text, cssClass, templateID = '') => TBApi.post('/api/selectflair', {
```

Comment:
`flairUser` has a default for `templateID` but `flairPost` doesn't. I don't think we need to default it to an empty string, do we? Letting it be `undefined` should be fine.
```suggestion
    TBApi.flairUser = (user, subreddit, text, cssClass, templateID) => TBApi.post('/api/selectflair', {
```

Reply:
I actually wasn't sure about that, so I put it in there just in case having it as `undefined` would cause any issues, but you're most probably right.